decimalSuite_test.go

```diff
@@ -78,6 +94,26 @@ func TestFromSuite(t *testing.T) {
 	}
 }
 
+func isInList(s string, list []string) bool {
+	for _, item := range list {
+		if item == s {
+			return true
+		}
+	}
+	return false
+}
+
+func setRoundingFromString(s string) Context64 {
+	switch s {
+	case "half_even":
+		return Context64{roundHalfEven}
+	case "half_up":
+		return Context64{roundHalfUp}
+	default:
```

Comment:
This should be more restrictive. Maybe `""` or `"default"` to denote _default_, which anything else triggering a `panic`.

Reply:
looking at it, the code doesn't even get here because there's a 'supportedRounding' that stops the code executing here anyway

Reply:
Codify this expectation by panicking.